packages/x-codemod/README.md

```diff
@@ -98,6 +98,24 @@ Renames the view components
 +<TimeClock {...props} />
 ```
 
+#### `view-components-rename-value-prop`
```

Comment:
We should also add a list item in the list above all sections (preset-safe) paragraph. 😉

Reply:
Added